doc/ref_cert/README.md

```diff
@@ -132,5 +132,5 @@ the network settings:
 
 <a name="available-cr"></a>
 ## Available Programs
-* [RC1 - LFN Based](lfn)
+* [RC1 - Openstack Based](lfn)
```

Comment:
Is it possible to rename the Github "lfn" directory to "RC1" so as to be consistent?

Reply:
OK, will do that.

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/1757/commits/3c622a3900b971577c30a01534848fef798e3815

Reply:
@rabi-abdel Thanks.